src/encoded/static/components/gene.js

```diff
@@ -140,10 +140,16 @@ class Gene extends React.Component {
                 </div>
 
                 <RelatedItems
-                    title={`Experiments targeting gene ${context.symbol}`}
+                    title={`Functional genomics experiments targeting gene ${context.symbol}`}
                     url={`/search/?type=Experiment&target.genes.uuid=${context.uuid}`}
                     Component={ExperimentTable}
                 />
+
+                <RelatedItems
+                    title={`Functional characterization experiments targeting gene ${context.symbol}`}
+                    url={`/search/?type=FunctionalCharacterizationExperiment&target.genes.uuid=$context.uuid}`}
```

Comment:
Missing an open brace before context.uuid, so this puts the literal string “...genes.uuid=$context.uuid}” as the query string.

Reply:
Thanks for catching this! I'll fix this asap.